src/librustc_errors/emitter.rs

```diff
@@ -1304,11 +1304,13 @@ impl EmitterWriter {
                 };
 
                 let column_width = if let Some(width) = self.terminal_width {
-                    width
+                    width.saturating_sub(code_offset)
                 } else if self.ui_testing {
                     140
                 } else {
-                    term_size::dimensions().map(|(w, _)| w - code_offset).unwrap_or(140)
+                    term_size::dimensions()
+                        .map(|(w, _)| w.saturating_sub(code_offset))
+                        .unwrap_or(std::usize::MAX)
```

Comment:
Why was this changed to usize::MAX?

Reply:
@bjorn3 If the dimensions aren't supplied and can't be queried, I decided that keeping the _current_ uncropped behavior makes more sense. By setting it to `MAX` it'll never crop.